eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/TranslationPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * File containing the TranslationPass class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Compiler;
+
+use eZ\Bundle\EzPublishCoreBundle\Translation\Collector;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * This compilation pass loads every ezplatform available translations into symfony translator.
+ */
+class TranslationPass implements CompilerPassInterface
```

Comment:
Given that we do not put several responsibilities in those compiler passes (they each do one particular thing), I'd still go for the more specific name.